Makefile

```diff
@@ -13,6 +13,7 @@ test_variant_app:
 base_addon_spec = spec/addons/base/**/*_spec.rb
 web_addon_spec = spec/addons/variants/web/**/*_spec.rb
 api_addon_spec = spec/addons/variants/api/**/*_spec.rb
+terraform_addon_spec = spec/addons/terraform/**/*_spec.rb
```

Comment:
I don't think we need this as the `terraform` could be located inside the base addon folder 🤔